dokku

```diff
@@ -207,14 +210,22 @@ case "$1" in
     ;;
 
   plugins)
-    ls -1 -d $PLUGIN_PATH/*/
+    for plugin in "$PLUGIN_PATH"/*/; do
```

Comment:
here I did this since I think outputting only the name of the plugins is more useful than the complete path. I might be wrong :sweat_smile: